marko.json

```diff
@@ -134,5 +134,11 @@
             "@role": "string"
         }
     },
-    "<ebay-pagination-item>": {}
+    "<ebay-pagination-item>": {},
+    "<ebay-checkbox>": {
+        "renderer": "./src/components/ebay-checkbox/index.js",
+        "@class": "string",
+        "@disabled": "boolean",
+        "@*": "expression"
```

Comment:
Please update this to use these (in this order, after the renderer, like the others):
```json
"@*": "expression",
"@html-attributes": "expression",
"@class": "string",
```